src/pudl/extract/eia861.py

```diff
@@ -29,13 +29,13 @@ def __init__(self, *args, **kwargs):
         self.METADATA = excel.Metadata("eia861")
         self.cols_added = []
         super().__init__(*args, **kwargs)
-
-    def process_raw(self, df, page, **partition):
-        """Rename columns with location."""
         warnings.warn(
             "Integration of EIA 861 into PUDL is still experimental and incomplete.\n"
             "The data has not yet been validated, and the structure may change."
         )
+
+    def process_raw(self, df, page, **partition):
+        """Rename columns with location."""
```

Comment:
because it was WARNING for every single table being extracted. which is many columns every time one attempts to access any of the 861 tables via the outputs.

Reply:
Ah okay, great! There's probably a similar thing happening in the `ferc714` extractor if you'd like to make the same change....